src/test/java/core/basesyntax/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,93 @@
+package core.basesyntax;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceImplTest {
+    private final RegistrationServiceImpl registrationService = new RegistrationServiceImpl();
+
+    @BeforeAll
+    public static void setUp() {
+        User validUser = new User();
+        validUser.setAge(18);
+        validUser.setLogin("Cherkassy");
+        validUser.setPassword("Qwerty3");
+        Storage.people.add(validUser);
+    }
+
+    @Test
+    public void register_NullOnFieldsOfUser_NotOk() {
```

Comment:
It's better to create separate tests instead of this test case
```
@Test
    void register_nullUser_notOk() {
        assertThrows(UserRegistrationException.class, () -> registrationService.register(null));
    }
```
```
register_passwordIsNull_notOk
register_loginIsNull_notOk
```